src/pages/auth/components/RegisterForm.jsx

```diff
@@ -82,8 +82,8 @@ export const RegisterForm = () => {
             style={{ marginTop: '8px' }}
             type='submit'
             fullWidth
-            rightIcon={<LoginIcon />}
-            title={t('Sign Up')}
+            rightIcon={isLoading ? <Spinner /> : <LoginIcon />}
```

Comment:
it's should be something like that
<Button loading={isLoading}

Then we will be able to show loader everywhere

Reply:
i add to button component new props "isLoading" and spinner